### ISSUE TITLE
buttons in table got AVT violation against Accessible name

### ISSUE THREAD

**Issue by lj7720** [IBM/carbon-components-angular#1744]
## Detailed description
> Describe in detail the issue you're having.
Buttons in table got AVT violation that "Accessible name does not match or contain the visible label text".

> Is this a feature request (new component, new icon), a bug, or a general issue?
Bug

> Is this issue related to a specific component?
table

> What did you expect to happen? What happened instead? What would you like to see changed?

> What browser are you working in?
Firefox 85.0.2

> What version of the Carbon Design System are you using?
carbon-components-angular: 3.54.2

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

## Steps to reproduce the issue
1. Open browser containing tables with buttons.
2. Run Accessibility checker.

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/74904663/109264402-e7133300-783f-11eb-817d-c8941bc1273a.png)
![image](https://user-images.githubusercontent.com/74904663/109264275-b0d5b380-783f-11eb-9a84-cb96a3828ca3.png)

**Comment by Donisius** [comment]
You should be able to use the `ariaSortLabel` property on `TableHeaderItem` or `CustomHeaderItem`  to set your own aria label if `data` isn't a string. I have fixed the documentation in #1760 to demonstrate this
